lib/rules/key-spacing.js

```diff
@@ -505,10 +506,11 @@ module.exports = {
          * @returns {int} Width of the key.
          */
         function getKeyWidth(property) {
+            const splitter = new GraphemeSplitter();
```

Comment:
Would this work well if we do `const splitter = new GraphemeSplitter();` only once, for example right after `require("grapheme-splitter")`, and always use the same instance? I'm getting ~20% better performance of this rule when linting this repo with `"key-spacing": ["error", { align: "value" }]` if a new instance is not created for each key vs if it is.

Reply:
Yes, we can move it up for better performance. Updated 👍🏻